config/routes.rb

```diff
@@ -118,6 +118,12 @@
     get "getting_started_completed" => :getting_started_completed
   end
 
+  resource :two_factor_authentication, only: %i(show create destroy) do
```

Comment:
Style/PercentLiteralDelimiters: `%i`-literals should be delimited by `[` and `]`.